hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -323,31 +322,54 @@ public void addTransactions(Map<Long, List<Long>> containerBlocksMap)
   public void close() throws IOException {
   }
 
+  private void getTransaction(DeletedBlocksTransaction tx,
```

Comment:
getTransaction -> getTransactions

Reply:
The function is used to fetch a single transaction so avoided s at end here.